Autor.php

```diff
@@ -0,0 +1,29 @@
+<?php
+declare(strict_types=1);
+
+class Autor {
+    private string $nombre;
+    private string $apellido;
+    private string $nacionalidad;
+    private int $anioNacimiento;
+
+    public function __construct(string $nombre, string $apellido, string $nacionalidad, int $anioNacimiento) {
+        $this->nombre = $nombre;
+        $this->apellido = $apellido;
+        $this->nacionalidad = $nacionalidad;
+        $this->anioNacimiento = $anioNacimiento;
+    }
+
+
+    public function getNombre(): string { return $this->nombre; }
```

Comment:
```suggestion
    public function getNombre(): string {
        return $this->nombre;
    }
```
Este formato es más legible, puede basarse en la guía de estilo de código de php [PSR-12](https://www.php-fig.org/psr/psr-12/) 4.4 Methods and Functions

Reply:
Después del cierre de métodos `}`.
Agregar un espacio en blanco.